docs/source/tutorials/experimentation/exp_setup.rst

```diff
@@ -0,0 +1,276 @@
+study, we will compare the following methods:
+
+* ``ASHA-TANH``, ``MOBSTER-TANH``: Runs ASHA and MOBSTER on the
+  simplified configuration space, where
+  ``hp_activation_fn_1 = hp_activation_fn_2 = "tanh"``
+* ``ASHA-RELU``, ``MOBSTER-RELU``: Runs ASHA and MOBSTER on the
+  simplified configuration space, where
+  ``hp_activation_fn_1 = hp_activation_fn_2 = "relu"``
+* ``ASHA``, ``MOBSTER``: Runs ASHA and MOBSTER on the original
+  configuration space
+* ``RS``, ``BO``: Runs baselines random search and Bayesian optimization on
+  the original configuration space
+
+Here is the script defining these alternatives:
+
+.. literalinclude:: ../../../../benchmarking/nursery/demo_experiment/baselines.py
+   :caption: benchmarking/nursery/demo_experiment/baselines.py
+   :start-after: # permissions and limitations under the License.
+
+* Different methods are defined in dictionary ``methods``, as functions
+  mapping ``method_arguments`` of type
+  :class:`~benchmarking.commons.baselines.MethodArguments` to a scheduler
+  object. Here, ``method_arguments.config_space`` contains the default
+  configuration space for the benchmark, where both
+  ``hp_activation_fn_1`` and ``hp_activation_fn_2`` are hyperparameters
+  of type ``choice(["tanh", "relu"])``.
+* For ``ASHA-TANH``, ``MOBSTER-TANH``, ``ASHA-RELU``, ``MOBSTER-RELU``, we fix
+  these parameters. This is done in ``_modify_config_space``, where
+  ``method_arguments.config_space`` is replaced by a configuration space where
+  the two hyperparameters are fixed (so methods do not search over them
+  anymore).
+* Another way to modify ``method_arguments`` just before a method is created,
+  is to use the ``map_extra_args`` argument of
+  :func:`~benchmarking.commons.hpo_main_simulator.main`, as detailed
+  `here <../benchmarking/bm_simulator.html#specifying-extra-arguments>`__. This
+  allows the modification to depend on extra command line arguments.
+
+Next, we define the benchmarks our study should run over. For our simple
+example, we use the ``fcnet`` benchmarks:
+
+.. literalinclude:: ../../../../benchmarking/nursery/demo_experiment/benchmark_definitions.py
+   :caption: benchmarking/nursery/demo_experiment/benchmark_definitions.py
+   :start-after: # permissions and limitations under the License.
+
+This is where you would have to plug in your own benchmarks, namely your training
+script with a bit of metadata. Examples are provided
+`here <../benchmarking/bm_local.html>`__ and
+`here <../benchmarking/bm_contributing.html>`__.
+
+Recording Extra Results
+-----------------------
+
+Next, we need to write the ``hpo_main.py`` script which runs a single experiment.
+As shown `here <../benchmarking/bm_simulator.html#defining-the-experiment>`__,
+this is mostly about selecting the correct ``main`` function among
+:func:`~benchmarking.commons.hpo_main_simulator.main`,
+:func:`~benchmarking.commons.hpo_main_local.main`,
+:func:`~benchmarking.commons.hpo_main_sagemaker.main`, depending on the trial
```

Comment:
This renders in such a way that you can't actually see which main is which:

![Screen Shot 2023-06-05 at 11 16 03 AM](https://github.com/awslabs/syne-tune/assets/1182047/5beb0ce4-7046-4453-9b04-4d1d2a539c49)